### ISSUE TITLE
[Feature] Add functionality to prevent cleaning on particular drives (Hot swap drive user)

### ISSUE THREAD

**Issue by Cutters1982** [stashapp/stash#314]
Hi folks

I'm sure I'm not the only person who doesn't have a NAS solution and has an archive of all porn spread across multiple drives (as it stands I have 13 drives, only 3 so far have I started archiving stuff from...)

My problem is, as I dont have a NAS, I have a hot swap drive bay on the front of my tower PC, so I disconnect one drive and put another in, if I then press the clean button it'll remove everything added from the drive that was just removed, which is obviously not something I want it to do.

I'd like to request the capability of adding drives/paths to an exclude list so that the clean command doesn't run on them, so as to prevent this issue from happening again. If I do need to delete something out then I would have to do it manually (I guess) via SQL DB Browser.

We aren't all in a position to go down the NAS route, so I'm sure there are other people with the same dilemma as me

Thanks!

**Comment by Leopere** [comment]
I'm not sure how NAS builds are associated with this feature request but from what I understand in your feature request you're looking for an ignore file/function?

**Comment by Cutters1982** [comment]
yeah, ability to block the delete function from working on a particular drive which has been disconnected, as I'm using a hotswap drive bay :)

**Comment by bnkai** [comment]
@Cutters1982 can we close this ? i think its a very specific use case that doesn't apply to almost anyone.

**Comment by Cutters1982** [comment]
As discussed on Discord, I think this is an important thing for people who hotswap to be able to have functionality on, so they can block a drive from being cleaned when disconnected, not everyone is in a fortunate enough position to have a NAS solution (I've now switched to NAS).
